gateway/README.md

```diff
@@ -233,11 +233,11 @@ For this, you will need to specify which port to bind to.  An example config:
 
 ##### Identify and Replace variables in Span Names
 
-The SignalFx listener has the ability to ideitify and replace variables in span names and turn them into tags. It uses go's named capture groups. See more at [https://golang.org/pkg/regexp/syntax/]. 
+The SignalFx listener has the ability to identify and replace variables in span names and turn them into tags. It uses regex-based replacement rules. See the <a target="syntax here" href="https://github.com/signalfx/gateway#graphite-dimensions">Click here to read more on Github about MetricDeconstructor options</a>
```

Comment:
What's up with the link? Destination seems wrong, and you can just use the Markdown syntax:

```markdown
See the [link name](url)
```